src/lifecycle/jvm_manager.cpp

```diff
@@ -92,33 +89,35 @@ void JvmManager::initialize_or_get_jvm(void* lib_handle, JvmUserConfiguration& u
 #endif
 
     jni::Jvm::initialize(java_vm, user_configuration.vm_type, jvm_options.version);
+    return true;
 }
 
-void JvmManager::initialize_jni_classes(jni::Env& p_env, ClassLoader* class_loader) {
+bool JvmManager::initialize_jni_classes(jni::Env& p_env, ClassLoader* class_loader) {
+    bool loaded_properly;
```

Comment:
Simplification:
instead of ```cpp
bool loaded_properly;
loaded_properly = TransferContext::initialize(p_env, class_loader);
loaded_properly = loaded_properly && TypeManager::initialize(p_env, class_loader);
...
```

We can write:
```cpp
bool loaded_properly {
    TransferContext::initialize(p_env, class_loader) &&
    TypeManager::initialize(p_env, class_loader) &&
    ...
};